pkg/db/task.go

```diff
@@ -0,0 +1,122 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+type Task struct {
+	ID      string `json:"id"`
+	Date    string `json:"date"`
+	Title   string `json:"title"`
+	Comment string `json:"comment"`
+	Repeat  string `json:"repeat"`
+}
+
+func AddTask(task *Task) (int64, error) {
+	query := `INSERT INTO scheduler (date, title, comment, repeat) VALUES (?, ?, ?, ?)`
+	res, err := db.Exec(query, task.Date, task.Title, task.Comment, task.Repeat)
+	if err != nil {
+		return 0, err
+	}
+	id, err := res.LastInsertId()
+	if err != nil {
+		return 0, err
```

Comment:
Хорошо бы оборачивать все ошибки
```suggestion
		return 0, fmt.Errorf("error getting last inserted id: %w", err)
```